### ISSUE TITLE
Makefile.meta should summarise status to console

### ISSUE THREAD

**Issue by jryans** [stephenrkell/liballocs#36]
At the moment running `Makefile.meta` may spew many millions of lines to the console as various subsidiary tools are invoked, leaving it a bit hard to follow what's actually happening.

It would be nice to default to a summarised view that prints just the current phase of work, and hides the detailed log unless you add `-v` or similar.

**Comment by stephenrkell** [comment]
I agree. Currently the -meta.c file gets its own gzipped log stored alongside it under /usr/lib/meta. It might make sense to follow that pattern everywhere, and use similar pretty-printing and ```make V=1``` that the Linux kernel build uses.

I have another branch, ```deep-static-allocs```, which (among other thins) is overhauling ```Makefile.meta``` in other ways. So I will probably leave this change until that is merged. But I will push my updates to that branch so that they hit this repo (it is taking me far longer to get them finished than I expected... mainly for lack of time, not complications).
